### DISCUSSION TITLE
What do the pull request of the Uptime repository have to do with Uptime of the monitored site?

### DISCUSSION THREAD

**Discussion by philoserf** [upptime/upptime#303] [General] [answered]
The changes recently to the uptime site pages display pull requests on the Uptime repo as Incidents. How are they relevant? I think they are not.

Example: https://philoserf.github.io/up/

An incident was created for a pull request on the Up repo. https://github.com/philoserf/up/pull/1

This is NOT desireable. The Up repo has nothing to do with the sites being monitored.

I believe this is a bug. Am I wrong?

**Comment by AnandChowdhary** [comment]
This looks interesting, I think Upptime is showing pull requests from your repo. I believe this is because the `/issues` endpoint of the GitHub API also returns pull requests (for some reason). I think we should filter on the frontend for non-PR issues? What do you think?

**Reply by philoserf** [reply]
I believe that only issues created in support of monitored sites (up/down) are relevant. No pull request would be relevant. It appears this behavior was introduced with planned maintenance, or when site build was extracted to its own repo. I haven't dug enough to be certain.

**Comment by philoserf** [comment, marked as answer]
Okay. Like magic, the condition is resolved.
